src/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -279,11 +215,9 @@ private void GetLinesFromStringBuilder()
                 _sb.Length = 0;
```

Comment:
The `StringBuilder` class exposes a `Clear()` method that is the equivollent of setting `Length` to zero, but is clearer in its intention.